src/Features/Core/Portable/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.ProjectState.cs

```diff
@@ -223,6 +224,8 @@ public bool IsEmpty(DocumentId documentId)
                 }
 
                 await SerializeAsync(serializer, project, result.ProjectId, _owner.NonLocalStateName, result.Others).ConfigureAwait(false);
+
+                AnalyzerABTestLogger.LogProjectDiagnostics(project, result);
```

Comment:
So we tracking project level error count ? Is this to study if the addition of vsix increased the count ? 
Can you add some comment pls